src/types/Attestation.ts

```diff
@@ -15,6 +16,10 @@ export default interface IAttestation {
   revoked: boolean
 }
 
+export interface IRevocableAttestation extends IAttestation {
+  witness: gabi.Witness | null
```

Comment:
Would be cool to remove the null here. The witness is private for the attester.  So Maybe remove the IRevocableAttestation interface. The attester would need the claimHash and the witness for revocation.

Not happy with that...